### ISSUE TITLE
Article: How to get the most out of your Visual Studio Code (VS)? Tips and tricks

### ISSUE THREAD

**Issue by Lisa-Holling** [tilburgsciencehub/website#1061]
I want to write an article that explains the basics of working with:
* Visual Studio Code
* Jupyter notebook
* Running python / R code
* Git integration with VS
* ...

Target group: mainly students/researchers who have some basic knowledge of VS, but want to use it more effectively

(similar to issue #1060 but then for VS)

**Comment by alexandervossen** [comment]
Good idea @Lisa-Holling! One thing that I feel is often overlooked is the "visual" part in VSCode.  Maybe you also find room to highlight a few of the great visual tools, such as indent-rainbow, vibrancy continued, vscode-icons, etc.

**Comment by Lisa-Holling** [comment]
@alexandervossen I am starting to work on this now, but was wondering where on the website would be an appropriate spot for this? Maybe somewhere around "Research Skills"? There's also going to be a similar article for Rstudio on this topic.

**Comment by alexandervossen** [comment]
great @Lisa-Holling ! I would maybe suggest under computer setup / software installation. We already have a RStudio Coding Sytle Guidelines block there so maybe the style guide fits there as well. A nice synergy could be  styling VSCode and combine with the new block on Positron that @valerievossen and @VirginiaMirabile are doing over here #1220 . Posit (the developers of Positron) claim that almost all plugins for VSCode work also for Positron. I am using it since two weeks now and I truely believe that it will replace RStudio in the not so long run.

**Comment by valerievossen** [comment]
It could also be nice in the IDE section of Computer set up? I cleaned up this section in a recent PR (still needs to be merged). Was thinking of adding the Positron article in there as well.
